pkg/deploy/strategy/rolling/rolling_test.go

```diff
@@ -18,6 +18,8 @@ import (
 )
 
 func TestRolling_deployInitial(t *testing.T) {
+	annotationUpdatePeriod = 1 * time.Millisecond
```

Comment:
not sure about this:) i think it's better to have separate constants for master and separate for tests